api/FetchEvent.json

```diff
@@ -318,25 +318,17 @@
               "version_added": "17",
               "notes": "NetworkError thrown if request mode is same-origin and response type is cors (see <a href='https://bugzil.la/1222008'>bug 1222008</a>). This is being worked on - see <a href='https://www.chromestatus.com/feature/5694278818856960'>https://www.chromestatus.com/feature/5694278818856960</a>."
             },
-            "firefox": [
-              {
-                "version_added": "59",
-                "notes": "NetworkError thrown if request mode is same-origin and response type is cors (see <a href='https://bugzil.la/1222008'>bug 1222008</a>)."
-              },
-              {
-                "version_added": "44",
-                "notes": "<a href='https://www.mozilla.org/en-US/firefox/organizations/'>Extended Support Releases (ESR)</a> before Firefox 78 ESR do not support service workers and the Push API."
-              }
-            ],
-            "firefox_android": [
-              {
-                "version_added": "59",
-                "notes": "NetworkError thrown if request mode is same-origin and response type is cors (see <a href='https://bugzil.la/1222008'>bug 1222008</a>)."
-              },
-              {
-                "version_added": "44"
-              }
-            ],
+            "firefox": {
+              "version_added": "44",
+              "notes": [
+                "Since Firefox 59, a <code>NetworkError</code> thrown if the request mode is <code>same-origin</code> and the response type is <code>cors</code> (see <a href='https://bugzil.la/1222008'>bug 1222008</a>).",
```

Comment:
Is this per-spec or not? If it's correct to throw `NetworkError` and that now happens, then no note at all is needed, because the old behavior is a subset of the new correct behavior. Nobody needs to do anything special to support the older version in this case.

Reply:
It looks like this is [per-spec behavior](urlhttps://github.com/whatwg/fetch/pull/655), but I wasn't sure since the note was added to Chromium browsers as well.  Maybe a subfeature would be appropriate here?

Reply:
If you can identify some behavior that web developers need to understand today, then maybe a subfeature would be appropriate. A test showing what kind of code triggers this `NetworkError` might help illuminate.